src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {App} from "./components/app/app.jsx";
+
+const unit = () => {
+  const settings = {
+    gameTime: 5,
+    errorCount: 3,
+  };
+
+  ReactDOM.render(
+      <App
+        errorCount={settings.errorCount}
+        gameTime={settings.gameTime}
+      />,
+      document.querySelector(`.main`)
+  );
+};
+
+unit();
```

Comment:
И тут поправить соответственно
```suggestion
init();
```